runtime-light/stdlib/server/args-functions.cpp

```diff
@@ -0,0 +1,90 @@
+// Compiler for PHP (aka KPHP)
+// Copyright (c) 2025 LLC «V Kontakte»
+// Distributed under the GPL v3 License, see LICENSE.notice.txt
+
+#include <cctype>
+#include <cstdint>
+#include <string_view>
+
+#include "runtime-common/core/runtime-core.h"
+#include "runtime-light/state/instance-state.h"
+#include "runtime-light/stdlib/server/args-functions.h"
+
+Optional<array<mixed>> f$getopt(const string &short_options, const array<string> &long_options,
+                                [[maybe_unused]] Optional<std::optional<std::reference_wrapper<string>>> rest_index) noexcept {
+  if (const auto &instance_st{InstanceState::get()}; instance_st.image_kind() != ImageKind::CLI) {
```

Comment:
Let's add `unlikely` attributes to all erroneous cases

Reply:
It is not erroneous case as for me. I think it's better to do image kind check constexpr in future